Clarify glossary entries for "simulator" (#2080)
modelica/fmi-standard: docs/3_3_model_exchange_example.adoc

=== Code Example

In the following example, the usage of the `fmi3XXX` functions is sketched in order to clarify the typical calling sequence of the functions in a simulator.

> Furthermore, it is assumed that one FMU is directly integrated in a simulator.

Reviewer: ```suggestion
Furthermore, it is assumed that one FMU is directly integrated in an importer.
```
